### DISCUSSION TITLE
WrappidDocs

### DISCUSSION THREAD

**Discussion by anantakumarghosh** [wrappid/.github#4] [Show and tell]
Checkout [WrappidDocs](https://wrappid.github.io/) - Documentation page for Wrappid 🤩

@wrappid/core-team 
@wrappid/testers

**Comment by techoneel** [comment]
Good initiative by @anantakumarghosh appreciate it.
But currently we have a [site](https//wrappid.dev) and the documentation can be driven by the github markdowns files.

So closing this discussion as of now.
